src/olympia/accounts/helpers.py

```diff
@@ -90,3 +90,7 @@ def generate_fxa_state():
 def camel_case(snake):
     parts = snake.split('_')
     return parts[0] + ''.join(part.capitalize() for part in parts[1:])
+
+
+def redirect_for_login(request):
+    return HttpResponseRedirect(default_fxa_login_url(request))
```

Comment:
So, we use the `.helpers` module usually to store template relevant code, e.g filters and jinja-functions but not general utilities (usually at least, there are some here). Can this move to `accounts.utils` instead?